### PR TITLE
Feature/event

### PR DESCRIPTION
**Pull request by docNord**
Addresses #7 

I think we could complement the list a bit, haven't listed every meeting we have had (I could possibly even find pictures from some of the other meetings), but for now at least I've included the events that are on the current web page.

### CODE REVIEW

**Comment by mgcth** [comment]
God damn... branch out from main 😢

**Comment by mgcth** [comment]
No don't close this. Doesn't help

**Comment by mgcth** [comment]
Let me fix it, 1 min.

**Comment by mgcth** [comment]
Could you please delete your local repo and pull a new one, just in case :)

File: content/posts/event-aw-2020-06-06.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+title: AW styrmedel för energi- och klimatmål
+date: 2020-06-06
+modified:
+author: Anders Nord
+category: Events
```
Comment (mgcth) [inline content/posts/event-aw-2020-06-06.md:5]:
```suggestion
category: Event
```

File: content/posts/event-aw-2020-10-07.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+title: AW kollektivt handlande
+date: 2020-10-07
+modified:
+author: Anders Nord
+category: Events
```
Comment (mgcth) [inline content/posts/event-aw-2020-10-07.md:5]:
```suggestion
category: Event
```

File: content/posts/event-aw-2022-03-16.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+title: AW med Frances Sprei
+date: 2022-03-16
+modified:
+author: Anders Nord
+category: Events
```
Comment (mgcth) [inline content/posts/event-aw-2022-03-16.md:5]:
```suggestion
category: Event
```

File: content/posts/event-aw-2022-04-08.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+title: Presentation av IPCC-delrapporten "Effekter, anpassning och sårbarhet"
+date: 2022-04-08
+modified:
+author: Anders Nord
+category: Events
```
Comment (mgcth) [inline content/posts/event-aw-2022-04-08.md:5]:
```suggestion
category: Event
```

**Review by mgcth — CHANGES_REQUESTED** [review]
We need some Swenglish.

**Comment by mgcth** [comment]
Ok, so you insisted on a new PR #12.

**Comment by docNord** [comment]
I redid the entire thing in another pull request rebased on the main version you have here which still carries the old stuff in the root. Sorry, didn't see your activity here.

**Pull request closed without merging**